addons/mail/static/tests/discuss/discuss_tests.js

```diff
@@ -1334,13 +1334,13 @@ QUnit.test(
 );
 
 QUnit.test(
-    "'Users' thread icon is displayed in top bar of channels of type 'group'",
+    "Thread avatar image is displayed in top bar of channels of type 'group'",
     async (assert) => {
         const pyEnv = await startServer();
         const channelId = pyEnv["mail.channel"].create({ channel_type: "group" });
         const { openDiscuss } = await start();
         await openDiscuss(channelId);
-        assert.containsOnce($, ".o-mail-Discuss-header .fa-users[title='Grouped Chat']");
+        assert.containsOnce($, ".o-mail-Discuss-content .o-mail-Discuss-threadAvatar");
```

Comment:
It's best to keep selector with `-header` rather than `-content` (it's more specific)